hazelcast/src/main/java/com/hazelcast/client/impl/protocol/ClientExceptionFactory.java

```diff
@@ -323,6 +325,7 @@ public ClientExceptionFactory(boolean jcacheAvailable, ClassLoader classLoader)
         register(NO_SUCH_FIELD_ERROR, NoSuchFieldError.class, ((message, cause) -> new NoSuchFieldError(message)));
         register(NO_SUCH_FIELD_EXCEPTION, NoSuchFieldException.class, ((message, cause) -> new NoSuchFieldException(message)));
         register(NO_CLASS_DEF_FOUND_ERROR, NoClassDefFoundError.class, ((message, cause) -> new NoClassDefFoundError(message)));
+        register(TASKLET_EXECUTION_ERROR, TaskletExecutionException.class, ((message, cause) -> new TaskletExecutionException(message)));
```

Comment:
It is not really needed after we have merged Jet and Hazelcast, but we were putting Jet-related exceptions into `com.hazelcast.jet.impl.util.ExceptionUtil#EXCEPTIONS`. Maybe we can do the same here to be consistent with others.